oidc-ui/README.md

```
@@ -0,0 +1,151 @@
+- `en.json` — English
+- `ar.json` — Arabic (RTL)
+- `hi.json` — Hindi
+- `kn.json` — Kannada
+- `km.json` — Khmer
+- `ta.json` — Tamil
+
+RTL languages are automatically detected and the `dir` attribute is set accordingly.
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)